src/components/SchedulePanel/components/RemoveHoursDialog/index.js

```diff
@@ -0,0 +1,62 @@
+import React, { Fragment } from 'react';
+import Dialog from '@material-ui/core/Dialog';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import DialogActions from '@material-ui/core/DialogActions';
+import Button from '@material-ui/core/Button';
+import { gordonColors } from '../../../../theme';
+
+export default class RemoveHoursDialog extends React.Component {
```

Comment:
Can we rename this component, since it is for all custom events, not just Office Hours?

Reply:
Will be done on next pull request